cmake/CMakeLists.txt

```diff
@@ -159,6 +159,10 @@ else (MSVC)
   set(LIB_PREFIX)
 endif (MSVC)
 
+if (UNICODE)
+  add_definitions(-DUNICODE -D_UNICODE)
```

Comment:
It's for the test coverage on continuous integration. CommandLineInterfaceTest.Win32ErrorMessage can still pass on ANSI without the fix. In appveyor.yml, UNICODE=ON is added.